test/e2e/suite/peered_vpc.go

```diff
@@ -262,6 +264,21 @@ func (t *PeeredVPCTest) NewVerifyPodIdentityAddon(nodeName string) *addon.Verify
 	}
 }
 
+// K8sClient returns the Kubernetes client for external package access
```

Comment:
What's the tradeoff in using getters vs exporting the fields? Since this is just tests, I'm not concerned with exporting the fields, in fact, I vote for doing that. Getters/setters have their place in Go but I don't think this is one of them.